src/garrysmod/gamemodes/zombiesurvival/gamemode/cl_scoreboard.lua

```diff
@@ -1,216 +1,249 @@
-	self.m_TitleLabel:SetTextColor(COLOR_GRAY)
-	self.m_TitleLabel:SizeToContents()
-	self.m_TitleLabel:NoClipping(true)
-	self.m_TitleLabel.Paint = BlurPaint
-
-	self.m_ServerNameLabel = vgui.Create("DLabel", self)
-	self.m_ServerNameLabel.Font = "ZSScoreBoardSubTitle"
-	self.m_ServerNameLabel:SetFont(self.m_ServerNameLabel.Font)
-	self.m_ServerNameLabel:SetText(GetHostName())
-	self.m_ServerNameLabel:SetTextColor(COLOR_GRAY)
-	self.m_ServerNameLabel:SizeToContents()
-	self.m_ServerNameLabel:NoClipping(true)
-	self.m_ServerNameLabel.Paint = BlurPaint
-
-	self.m_AuthorLabel = EasyLabel(self, "by "..GAMEMODE.Author.." ("..GAMEMODE.Email..")", "ZSScoreBoardPing", COLOR_GRAY)
-	self.m_ContactLabel = EasyLabel(self, GAMEMODE.Website, "ZSScoreBoardPing", COLOR_GRAY)
-
-	self.m_HumanHeading = vgui.Create("DTeamHeading", self)
-	self.m_HumanHeading:SetTeam(TEAM_HUMAN)
-
-	self.m_ZombieHeading = vgui.Create("DTeamHeading", self)
-	self.m_ZombieHeading:SetTeam(TEAM_UNDEAD)
-
-	self.m_PointsLabel = EasyLabel(self, "Score", "ZSScoreBoardPlayer", COLOR_GRAY)
-	self.m_RemortCLabel = EasyLabel(self, "R.LVL", "ZSScoreBoardPlayer", COLOR_GRAY)
-
-	self.m_BrainsLabel = EasyLabel(self, "Brains", "ZSScoreBoardPlayer", COLOR_GRAY)
-	self.m_RemortCZLabel = EasyLabel(self, "R.LVL", "ZSScoreBoardPlayer", COLOR_GRAY)
-
-	self.ZombieList = vgui.Create("DScrollPanel", self)
-	self.ZombieList.Team = TEAM_UNDEAD
-
-	self.HumanList = vgui.Create("DScrollPanel", self)
-	self.HumanList.Team = TEAM_HUMAN
-
-	self:InvalidateLayout()
+    self.NextRefresh = RealTime() + 0.1
+
+    self.m_Header = vgui.Create("Panel", self)
+    self.m_Header:Dock(TOP)
+    self.m_Header:DockPadding(16, 0, 16, 0)
+
+    self.m_Header.Paint = function(self, w, h)
+        surface.SetDrawColor(5, 5, 5, 220)
+        PaintGenericFrame(self, 0, 0, w, h, 32)
+    end
+
+    self.m_TitleLabel = vgui.Create("DLabel", self.m_Header)
+    self.m_TitleLabel:SetFont("ZSScoreBoardSubTitle")
+    self.m_TitleLabel:SetText(string.format("%s | %s", GAMEMODE.Name, translate.Get(game.GetMap())))
+    self.m_TitleLabel:SetTextColor(COLOR_GRAY)
+    self.m_TitleLabel:Dock(LEFT)
+    self.m_TitleLabel:SetContentAlignment(5)
+    self.m_TitleLabel:SizeToContents()
+
+    self.MapChangesLabel = vgui.Create("DLabel", self.m_Header)
+    self.MapChangesLabel:SetFont("ZSScoreBoardSubTitle")
+    self.MapChangesLabel:SetTextColor(COLOR_GRAY)
+    self.MapChangesLabel:Dock(RIGHT)
+    self.MapChangesLabel:SetContentAlignment(5)
+    self.MapChangesLabel:SizeToContents()
+
+    local texRightEdge = surface.GetTextureID("gui/gradient")
+    local texCorner = surface.GetTextureID("zombiesurvival/circlegradient")
+    local texDownEdge = surface.GetTextureID("gui/gradient_down")
+
+    self.grid = vgui.Create("ZSGrid", self)
+    self.grid:Dock(FILL)
+    self.grid:SetColumns(2)
+    self.grid:SetHorizontalMargin(16 * BetterScreenScale())
+    self.grid:SetVerticalMargin(16 * BetterScreenScale())
```

Comment:
Create a grid for the two columns of players. Before, these columns were created and sized / positioned by hand. Now, it's all calculated to fill the parent element space.